package.json

```diff
@@ -3,29 +3,29 @@
   "version": "4.0.4",
   "author": "Johannes Ewald @jhnns",
   "description": "Less loader for webpack. Compiles Less to CSS.",
-  "main": "dist/index.js",
+  "main": "dist/cjs.js",
   "scripts": {
     "create-spec": "babel-node test/helpers/createSpec.js",
     "pretest": "npm run create-spec",
```

Comment:
The `create-spec` (`pretest`) `script` doesn't seem to run on the `PART=coverage` build https://travis-ci.org/webpack-contrib/less-loader/jobs/244975479